### PR TITLE
Haddock and changelog documentation

### PR DESCRIPTION
**Pull request by OlivierSohn**

### CODE REVIEW

**Comment by OlivierSohn** [comment]
Maybe now would be a good time to push the package to hackage? I can help on that side if needed.

File: README.txt [hunk 1]
```diff
@@ -21,13 +21,13 @@ or simply:
...
        cabal install
 
-This will install a PortMidi package that contains a Sound.PortMidi module. 
+This will install a PortMidi package that contains a Sound.PortMidi module.
 
 ===================
 Contact Information
 ===================
 
-You may send your bug report and feature request to the package
```
Comment (ninegua) [inline README.txt]: I think this paragraph can be changed to point to github repo: https://github.com/ninegua/PortMidi

**Comment by ninegua** [comment]
Thanks! Looking good! Have you tested compilation on different platforms?

Since you are a maintainer now, I suppose you can also submit to Hackage? Maybe you can give it a try first, and I can help if it does work out.

**Comment by OlivierSohn** [comment]
Reading the [hackage doc](http://hackage.haskell.org/upload) :

> If you upload a package or package candidate and no other versions exist in the package database, you become part of the maintainer group for that package, and you can add other maintainers if you wish. If a maintainer group exists for a package, only its members can upload new versions of that package.

It seems you need to add me as a maintainer because it's an existing package.

> Have you tested compilation on different platforms?

I have tested on my setup, with ghc 8.2.2.

Is there a list of officially supported GHC versions?

To test other GHC versions / platforms I'd recommend using automated CI for that, like [travis](https://travis-ci.org/) for linux and [appveyor](https://www.appveyor.com/) for windows. They both use the yml format to describe the build environments, I use them personally in [this project](https://github.com/OlivierSohn/hamazed) (see the `.travis.yml` and `appveyor.yml` files). But for this to work, the repo owner has to link the repo to travis and appveyor first.

File: README.txt (hunk 1 above)
Comment (OlivierSohn) [reply README.txt]: Done

**Comment by OlivierSohn** [comment]
@ninegua I'm going to merge this, as it's only doc changes.

Concerning building on other platforms, I don't see what change would make a platform-specific failure, but I might be wrong... Did you have something in mind? Older GHCs?

**Pull request merged**